registered/intervals/query.py

```diff
@@ -39,12 +39,24 @@ def read_database(
   0 AS IntervalType,
   gn1.geo_node_abbr AS FromStopNumber,
   gn1.geo_node_name AS FromStopDescription,
-  gn1.mdt_latitude / 10000000 AS FromStopLatitude,
-  gn1.mdt_longitude / 10000000 AS FromStopLongitude,
+  CASE
+        WHEN gn1.use_survey = 1 THEN (gn1.latitude / 10000000)
+        ELSE (gn1.map_latitude / 10000000) END
+  as FromStopLatitude,
```

Comment:
Is it possible to pull out the 10000000?
`(CASE WHEN gn1.use_survey = 1 THEN g1.latitude ELSE gn1.map_latitude) / 10000000`

Reply:
Ok, I made this change.